08-calculator_project/calculator.java

```diff
@@ -0,0 +1,74 @@
+import java.util.*;
+
+public class calculator {
+  public static void main(String[] args) {
+    Scanner input = new Scanner(System.in);
+    System.out.print("Enter the fractions expression: ");
+    String exp = input.nextLine();
+    produceAnswer(exp);
+
+  } //end main
+
+  public static String convertFraction(String fraction) {
+    int lowDash = fraction.indexOf("_");
+    int divide = fraction.indexOf("/");
+    String num1 = fraction.substring(0, lowDash);
+    String num2 = fraction.substring(lowDash + 1, divide);
+    String num3 = fraction.substring(divide + 1, fraction.length());
+    int cal = Integer.parseInt(num1) * Integer.parseInt(num3) + Integer.parseInt(num2);
+    String num = reduceFraction(cal, Integer.parseInt(num3));
+    return num;
+  } //end determineFraction()
+
+  public static void produceAnswer(String exp) {
```

Comment:
this should be a non-void method and whatever the return is should be printed in `main`